public/src/world/stage.js

```
@@ -0,0 +1,509 @@
+    }
+  }
+
+  checkClearCondition() {
+    if (this.isCleared) return true;
+
+    let progress = 0;
+    let isNearClear = false;
+
+    switch (this.clearCondition.type) {
+      case StageClearConditions.KILL_ALL_ENEMIES:
+        const totalEnemies = this.enemies.length;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)